pretext/project/__init__.py

```diff
@@ -505,7 +510,7 @@ class Project(pxml.BaseXmlModel, tag="project"):
     on the disk, and Paths for where to build output and maintain a site.
     """
 
-    ptx_version: t.Optional[str] = pxml.attr(name="ptx-version")
+    ptx_version: t.Literal["2"] = pxml.attr(name="ptx-version")
```

Comment:
I switched this because I was getting a validation error when trying to run `pretext support` using a legacy `project.ptx` (in fact, even if I put `ptx-version="2"` I got the error). 

However, given that we check for this later, do we need this line at all?

Reply:
That's a bug in my code -- what sort of error did you get? Or how can I reproduce it? When I run `pretext support` on a project just created by `pretext new`, everything is OK.

Reply:
You are right.  It must have been because of another thing that hadn't been fixed previously that this was throwing the error.